cypress/integration/pages/articles.js

```diff
@@ -11,6 +12,7 @@ const serviceHasFigure = service =>
 const serviceHasCaption = service => service === 'news';
 // TODO: Remove after https://github.com/bbc/simorgh/issues/2962
 const serviceHasCorrectlyRenderedParagraphs = service => service !== 'sinhala';
+const serviceHasTimestamp = service => service === 'news';
```

Comment:
can't we re-use `serviceHasCaption` ? seems to do the same thing

Reply:
`serviceHasTimestamp` looks for the timestamp on articlePage
`serviceHasCaption` looks for image/video captions